sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleCheckDocumentStatuses.cs

```diff
@@ -0,0 +1,85 @@
+﻿// <copyright file="SampleCheckDocumentStatuses.cs" company="Microsoft Corporation">
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+
+using Azure;
+using Azure.AI.Translation.Document;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+/**
+FILE: SampleCheckDocumentStatuses.cs
+DESCRIPTION:
+    This sample demonstrates how to create a translation job and then monitor each document's status
+    and progress within the job.
+USAGE:
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+    3) AZURE_SOURCE_CONTAINER_URL - the container SAS URL to your source container which has the documents
+        to be translated.
+    4) AZURE_TARGET_CONTAINER_URL - the container SAS URL to your target container where the translated documents
+        will be written.
+**/
+
+namespace DocumentTranslatorSamples
+{
+    class SampleCheckDocumentStatuses
```

Comment:
with small differences, looks like this is the same as: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_DocumentStatusAsync.cs

Maybe we should modify that one to add the extra printing for when there is an error, or show the translation progress for running operations

Reply:
Removed (Duplicate)